modules/seqexec/model/src/main/scala/seqexec/model/ExecutionQueueView.scala

```diff
@@ -7,17 +7,24 @@ import cats.Eq
 import cats.implicits._
 import gem.Observation
 import monocle.macros.Lenses
-import seqexec.model.enum.{BatchCommandState, BatchExecState}
+import seqexec.model.enum.BatchCommandState
+import seqexec.model.enum.BatchExecState
```

Comment:
Why separate these? Is this a style thing?

Reply:
yes, otherwise I easily have to split the import into several lines, this is handled automatically